lib/core/util/dimensions.dart

```diff
@@ -1,5 +1,49 @@
+import 'package:flutter/material.dart';
+
 abstract class Dimensions {
```

Comment:
Review this file, try to keep here only the constants that are used in more than one place, and leave the specific const inside each file that uses it

Reply:
Done, moved some constants to their respective files. There might still be some that are used in one place but that's because I expect to use them in other places in the future.